main/core/src/mill/eval/Evaluator.scala

```diff
@@ -659,44 +666,27 @@ object Evaluator {
   def plan(rootModule: BaseModule, goals: Agg[Task[_]]) = {
```

Comment:
Looks like we no longer need that rootModule parameter here?

Reply:
Good catch, seems like we used to use it to try and resolve the "final" implementation of a task to do override count comparisons. Now we are simply finding it via the topo sort, it's no longer necessary